packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -143,4 +145,49 @@ async function getLocalDebugConfig(key: string): Promise<string | undefined> {
 
 export async function getSkipNgrokConfig(): Promise<string | undefined> {
   return getLocalDebugConfig(constants.skipNgrokConfigKey);
-}
\ No newline at end of file
+}
+
+async function getPortListeningPidWindows(host: string, port: number): Promise<string | undefined> {
+  try {
+    let command = `(Get-NetTCPConnection -LocalPort ${port} -State Listen).OwningProcess`;
```

Comment:
Try to use existing npm package first, e.g., https://www.npmjs.com/package/tcp-port-used, https://www.npmjs.com/package/portfinder, etc...

Reply:
Great solution. I suppose it will be more stable than shell solution. I investigated several npm package, and decide to use `net.createServer` and `net.listen` directly. The reason is that these package cannot handle one thing like, node inspector listens on `127.0.0.1`, it is OK if there is already a process listens on `0.0.0.0`, `[::]` or `[::1]` on the same port.